driverthroughput/main.py

```diff
@@ -2,29 +2,40 @@
 from driverprocessresults.main import *
 
 if __name__ == "__main__":
+    """ Takes the pwd, then navigates to the root to append packages.
+    Python is then able to find our *.py files in that directory.
+    """
     sys.path.append("..")
 
+if __name__ == "__main__":
```

Comment:
Here, too, you have duplicate `if __name__ == "__main__":`

Reply:
answered above!